docs/querying/sql-data-types.md

```diff
@@ -85,8 +87,45 @@ the `UNNEST` functionality available in some other SQL dialects. Refer to the do
 > they are handled in Druid SQL and in native queries. For example, expressions involving multi-value dimensions may be
 > incorrectly optimized by the Druid SQL planner: `multi_val_dim = 'a' AND multi_val_dim = 'b'` is optimized to
 > `false`, even though it is possible for a single row to have both "a" and "b" as values for `multi_val_dim`. The
-> SQL behavior of multi-value dimensions will change in a future release to more closely align with their behavior
-> in native queries.
+> SQL behavior of multi-value dimensions may change in a future release to more closely align with their behavior
+> in native queries, but the [multi-value string functions](./sql-multivalue-string-functions.md) should be able to provide
+> nearly all possible native functionality.
+
+## Arrays
+Druid supports `ARRAY` types constructed at query time, though it currently lacks the ability to store them in
+segments. `ARRAY` types behave as standard SQL arrays, where results are grouped by matching entire arrays. This is in
+contrast to the implicit `UNNEST` that occurs when grouping on multi-value dimensions directly or when used with the
+multi-value functions. You can convert multi-value dimensions to standard SQL arrays either by explicitly converting
+them with `MV_TO_ARRAY`, or implicitly when used within the [array functions](./sql-array-functions.md). Arrays may
+also be constructed from multiple columns using the array functions.
+
+## Multi-value strings behavior
+The behavior of Druid [multi-value string dimensions](multi-value-dimensions.md) varies depending on the context of their usage.
+
+When used with standard `VARCHAR` functions, which expect a single input value per row, Druid maps the function across all values in the row.
+the function across all values in the row. If the row is null or empty, the function will recieve `NULL` as its input,
```

Comment:
```suggestion
If the row is null or empty, the function will receive `NULL` as its input,
```

Reply:
```suggestion
the function across all values in the row. If the row is null or empty, the function receives `NULL` as its input,
```

Reply:
Try to avoid future tense when possible.